requirements.txt

```diff
@@ -1,22 +1,22 @@
-django>=2.0,<2.1
-django-registration-redux>=2.2,<2.3
-djangorestframework>=3.7,<3.8
-django-oauth-toolkit>=1.0,<1.1
-django-cors-headers>=2.1,<2.2
-requests>=2.18,<2.19
-django-filter>=1.1,<1.2
+django>=2.1,<2.2
+django-registration-redux>=2.4,<2.5
+djangorestframework>=3.8,<3.9
+django-oauth-toolkit>=1.2,<1.3
+django-cors-headers>=2.4,<2.5
+requests>=2.19,<2.20
+django-filter>=2.0,<2.1
 django-bootstrap3>=8.0<9.0
 time_intervals<2.0
 WeasyPrint>=0.42<0.43
 PyPDF2>=1.26,<1.27
-elasticsearch>=6.0,<6.1
-numpy>=1.14, <1.15
+elasticsearch>=6.3,<6.4
```

Comment:
We are still on elasticsearch 5. The python elasticsearch library is major versioned in line with the version of elasticsearch you are connecting to, so this should be >=5.0,<6.0. I have no idea why it was already 6 but that is bad.
https://elasticsearch-py.readthedocs.io/en/master/#compatibility

Reply:
Pinned it to 5